docs/reference/builder.md

```diff
@@ -152,18 +152,22 @@ Example (parsed representation is displayed after the `#`):
     ADD . $foo       # ADD . /bar
     COPY \$foo /quux # COPY $foo /quux
 
-The instructions that handle environment variables in the `Dockerfile` are:
+Environment variables are supported by the following list of instructions in 
+the `Dockerfile`:
 
-* `ENV`
 * `ADD`
 * `COPY`
-* `WORKDIR`
+* `ENV`
 * `EXPOSE`
-* `VOLUME`
+* `ONBUILD` (see note)
```

Comment:
I don't think we need the `(see note)` here. The note is only for users running `< 1.4` which is quite ancient in Docker terms.